assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -314,7 +314,6 @@ class PageEdit extends Component {
 				<InspectorControls>
 					<PanelColorSettings
 						title={ __( 'Background Color', 'amp' ) }
-						initialOpen={ false }
```

Comment:
Can we do the same for regular blocks as well?

This is bugging me a bit:

https://github.com/ampproject/amp-wp/blob/c894d9a6bdb4d8d2ccf73d1ef58f8a73a6b76e1a/assets/src/stories-editor/components/higher-order/with-amp-story-settings.js#L471-L486

Reply:
This should be another issue / PR